### PR TITLE
cutadapt: do not ignore maxn for 0 and adapter sanitizer

### PR DESCRIPTION
**Pull request by bernt-matthias**
in addition it should fix https://github.com/galaxyproject/tools-iuc/issues/2986 

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/cutadapt/macros.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         #end for
         #for $aa in $library.r1.anywhere_adapters
             #if $aa.anywhere_adapter_source.anywhere_adapter_source_list == 'builtin':
-                -b ${aa.anywhere_adapter_source.anywhere_adapter.fields.name}"='${aa.anywhere_adapter_source.anywhere_adapter}'
+                -b "${aa.anywhere_adapter_source.anywhere_adapter.fields.name}"='${aa.anywhere_adapter_source.anywhere_adapter}'
```
Comment (bgruening) [inline tools/cutadapt/macros.xml:20]: can this be single-quoted?

Comment (bernt-matthias) [reply tools/cutadapt/macros.xml:20]: probably

Comment (bernt-matthias) [reply tools/cutadapt/macros.xml]: certainly .. a good idea.

**Comment by bgruening** [comment]
Thanks!

**Comment by bernt-matthias** [comment]
Thanks @bgruening . Can you trigger upload to the TS or ping someone who can?

**Comment by nsoranzo** [comment]
Manually uploaded to the TS.

**Comment by bernt-matthias** [comment]
Thanks a lot @nsoranzo

**Pull request merged**